build-ps/debian/rules

```diff
@@ -174,7 +174,11 @@ override_dh_auto_install:
 	(cd $(galeradir) && cp packages/rpm/README $(TMP)/)
 	mkdir -p $(TMP)/usr/bin/pxc_extra/
 	(cp -r pxb-2.4 $(TMP)/usr/bin/pxc_extra/)
+	(mv $(TMP)/usr/bin/pxc_extra/pxb-2.4/lib/xtrabackup/* $(TMP)/usr/bin/pxc_extra/pxb-2.4/lib/)
+	(rm -rf $(TMP)/usr/bin/pxc_extra/pxb-2.4/lib/xtrabackup)
```

Comment:
path spec with $(TMP) should be quoted to avoid potential problems with $(TMP) values that may contain spaces. $(TMP) is based on $(CURDIR) (line 21) which could potential contain spaces. This could cause problems, especially with rm -rf!